getting-started/index.html

```diff
@@ -341,21 +341,20 @@ <h2>More information</h2>
 			}
 			cmdlist.add(partial)
 		})
-		if (cmdlist.size() &gt; 0) {
+		if (cmdlist.size() > 0) {
 			jQuery('#relcommands').removeClass('hide')
 		}
 	})
 	jQuery('#cmdlist').on('click', '.CommandName', function(e) {
 		e.preventDefault();
-		document.location.href = 'https://dbatools.io/functions/' + jQuery(this).text();
+		document.location.href = 'https://dbatools.io/functions' + jQuery(this).text();
 	})
 	jQuery('#search-ft').on('keyup', function() {
-  		var searchString = jQuery(this).val();
-  		cmdlist.search(searchString);
+	var searchString = jQuery(this).val();
+	cmdlist.search(searchString);
 	})
-
-
-			</div>
+	</script>
+</div>
```

Comment:
This was primary fix for it that I could figure out. Syntax in the script itself I guess when it was converted from WP replaced the `>` with the `&gt;`, and then adding the closing script and div tag.